msteams-platform/sbs-bots-with-sso.yml

```diff
@@ -131,15 +131,14 @@ items:
     
         1. Select your **Resource group** from the dropdown list that you want to provision for your bot app.
         
-         
            You can also create a new resource group by the following steps:
 
               1. To create new resource group, select **Create new** > enter resource name > select **OK**.
 
                    :::image type="content" source="~/assets/images/bots/newresourcecreation.png" alt-text="New resource creation":::
 
               1. If you've created a new resource group, select the required location from **New resource group location** dropdown list.
-        
```

Comment:
Not sure but it seems that the article doesn't have an edit except inserting this space. Check if this update was required. Remove, if not required.

Reply:
There is no extra space which needs to be removed, there is an image after the text, hence the space, which cannot be removed. Please advise.